index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="./css/index.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <p>Hello World</p>
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <div class="nav">
+      <ul>
+        <li class="listitems">
+          <a href="#">About</a>
+        </li>
+        <li class="listitems">
+          <a href="#">work</a>
+        </li>
+        <li class="listitems">
+          <a href="#">contact</a>
+        </li>
+      </ul>
+    </div>
+    <!-- start sectionone -->
+    <section class="sectionone pd-y" id="sectionone">
+      <div class="sectionone__wrapper">
+        <h1 class="" id="">Hey I am Mimic</h1>
+      <small><h3 class="center">a web developer</h3></small>
```

Comment:
Please note that the small element should not contain an h3 element. Instead, you should have an h1 element outside of the small element because the h3 element should be outside of small element because it will help with the concept of semantic HTML for a web page best practice, Which will increase your website SEO.

Reply:
You have nicely explained the purpose of using semantic tags. But I feel like, the purpose of using a small tag is a little bit different. Kindly take a look at this [link](https://www.geeksforgeeks.org/html-small-tag/). While discussing technical things, it is better to look for the right documentation online and share that with other developers so that they might look that in detail with some context.